awscli/customizations/emr/helptext.py

```diff
@@ -205,7 +205,8 @@
     ' are used for the core node type.</p>')
 
 ADDITIONAL_INFO = (
-    '<p>Specifies additional information during cluster creation.</p>')
+    '<p>Specifies additional information during cluster creation. To set development mode when starting your EMR cluster,'
+    ' set this parameter to <code>'{"clusterType":"development"}'</code>.</p>')
```

Comment:
My guess is that the tests are not passing because there are single quotes around the parameter value. I think we will need to escape these quotes (e.g. `\'`). So update the value in the string to:
```
' set this parameter to <code>\'{"clusterType":"development"}\'</code>.</p>'
or we can just remove the single quote altogether:
```
' set this parameter to <code>{"clusterType":"development"}</code>.</p>'
```
```

Reply:
Thanks, I made a second commit hopefully resolving the issue.